diff -up SDL_sound-1.0.3/Makefile.am.omv~ SDL_sound-1.0.3/Makefile.am
--- SDL_sound-1.0.3/Makefile.am.omv~	2020-01-29 13:57:32.627727515 +0100
+++ SDL_sound-1.0.3/Makefile.am	2020-01-29 13:57:40.014727840 +0100
@@ -32,7 +32,7 @@ libSDL_sound_la_LDFLAGS = 		\
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 libSDL_sound_la_LIBADD =	\
 	decoders/libdecoders.la	\
-	$(TIMIDITY_LIB) $(MPGLIB_LIB)
+	$(TIMIDITY_LIB) $(MPGLIB_LIB) -lm
 
 EXTRA_DIST =	\
 	CREDITS \
